lib/openstax/aws/stack.rb

```diff
@@ -297,7 +297,16 @@ def self.format_hash_as_stack_parameters(params={})
     end
 
     def status
-      aws_stack.stack_status
+      begin
+        aws_stack.stack_status
+      rescue Aws::CloudFormation::Errors::ValidationError => ee
```

Comment:
minor.  https://github.com/rubocop-hq/ruby-style-guide shows rescue using a shorthand convention of 'e' over 'ee'

Reply:
I can't do single letter variable names :-)